include/stronk/stronk.h

```diff
@@ -504,4 +595,16 @@ struct can_index : can_const_index<StronkT>
     }
 };
 
+template<typename StronkT>
+struct transform_skill
```

Comment:
Choose a different kind of name, because its not just forwarding transform to the underlying type - which all the other skills does. What would be a good name for this? And should this just always be part of the base stronk type?

Reply:
Hmm. Why would can_transform not work?
In terms of e.g. std::optional this of course needs to return underlying_type such that the type is fixed.
I think it is fine to include by default.

Reply:
Skills like, can_size just calls size on the underlying value, transform here is not so much a skill as the monoid functionality -  so including it in base class helps me not have to care about this which is nice

Reply:
So exactly like you say, for std::optional, normal skills just call: std::optional<int>{}.transform(func) Whereas this will call transform(std::optional<int>{})